torch/optim/_functional.py

```diff
@@ -206,22 +209,16 @@ def adadelta(params: List[Tensor],
     See :class:`~torch.optim.Adadelta` for details.
     """
 
-    for (param, grad, square_avg, acc_delta) in zip(params, grads, square_avgs, acc_deltas):
-        if weight_decay != 0:
-            grad = grad.add(param, alpha=weight_decay)
+    adadelta_fn(params,
```

Comment:
I think the doc here can be moved to the one in adadelta.py and the only thing left in this file will be `from .adadelta import adadelta`